docs/src/examples/spindle.adoc

```diff
@@ -78,9 +77,9 @@ net spindle-enable spindle.0.on => parport.0.pin-14-out
 
 == Spindle Direction(((Spindle Direction Example)))
 
-If you have direction control of your spindle the HAL pins
-spindle.N.forward and spindle.N.reverse are controlled by M3
-and M4. Spindle speed 'Sn' must be set to a positive non-zero value for
+If you have direction control of your spindle, then the HAL pins
+'spindle.N.forward' and 'spindle.N.reverse' are controlled by the G-codes
```

Comment:
Historically, I think we usually typeset pin names in bold, and sometimes showed the variable parts in italics. At least, this is what `halcompile`-generated manpages do.

Reply:
I typically left all the quotes as they came with the English. No idea why I added them here, likely because how these were presented before and after. There are quite some occurences also of backticks in other files we should somehow revisit.

Reply:
I suggest to leave those quotes in until the style guide has an opinion.